### ISSUE TITLE
[DOC] IPAM CR usage and multiple-CIS use case

### ISSUE THREAD

**Issue by alonsocamaro** [F5Networks/k8s-bigip-ctlr#3345]
#### Title
[DOC] IPAM CR usage and multiple-CIS use case

#### Description

It would be convenient to document the following in the flow-chart for CIS-FIC:

- indicate a text description indicating that the CIS creates the CR always in the kube-system namespace, independently of where the CIS and IPAM instances are deployed
- indicate explicitly that CIS and FC need to be in the same namespace directory
- indicate explicitly that if it is desired to share FC amongst several CIS instances, for the same or different BIG-IPs, these all need to be in the same namespace
- The diagram doesn´t show the right name for the F5IPAM resource created by CIS, it has the following format instead: ``<deployment>.<namespace>.ipam`` as shown next

```
[cloud-user@ocp-provisioner routes-bigip]$ oc get ipams.fic.f5.com -A
NAMESPACE     NAME                                         AGE
kube-system   k8s-bigip1-ctlr-deployment.sc-twotier.ipam   5m35s
```

**Comment by alonsocamaro** [comment]
Additionally, it is not possible to have to FIC instances per cluster

**Comment by trinaths** [comment]
Created [CONTCNTR-4653] for internal tracking.

**Comment by trinaths** [comment]
Fixed in 2.16.1
